static/app/utils/theme.tsx

```diff
@@ -929,7 +929,7 @@ const commonTheme = {
 
     // We have an array that maps `number + 1` --> list of `number` colors
     getColorPalette: (length: number) =>
-      CHART_PALETTE[Math.min(CHART_PALETTE.length - 1, length + 1)]!,
+      CHART_PALETTE[Math.min(CHART_PALETTE.length - 1, length + 1)],
```

Comment:
instead of putting ?? [] everywhere
```
getColorPalette: (length: number) =>
      CHART_PALETTE[Math.min(CHART_PALETTE.length - 1, length + 1)] ?? [],
```

Reply:
I didnt want to change that API because we were never checking for indexed access before and idk if there'a any instances that might actually be checking for this